### PR TITLE
Move the seed node back to single 1GbE NIC

### PR DESCRIPTION
**Pull request by paulbrowne**
Move all of the seed node's networks back to a single 1GbE interface, rather than the 2x 25GbE NICs that were being used for admin, OC provision, OC OoB, internal and iLab networks.

### CODE REVIEW

File: etc/kayobe/inventory/group_vars/seed/network-interfaces [hunk 1]
```diff
@@ -14,16 +16,16 @@ bond0_net_bond_slaves:
 # Admin network to rest of control plane
 # NOTE: seed node is the default gateway for all other control nodes
 #       via this network, and all other nodes access this via 1GbE
-admin_oc_net_interface: "{{ bond0_net_interface }}.{{ admin_oc_net_vlan }}"
+admin_oc_net_interface: "{{ 1gbe_physical_interface }}.{{ admin_oc_net_vlan }}"
 
 # Overcloud provisioning network
 # NOTE: all other nodes access this via 1GbE for PXE boot
-provision_oc_net_interface: "{{ bond0_net_interface }}.{{ provision_oc_net_vlan }}"
+provision_oc_net_interface: "{{ 1gbe_physical_interface }}.{{ provision_oc_net_vlan }}"
```
Comment (JohnGarbutt) [inline etc/kayobe/inventory/group_vars/seed/network-interfaces]: What about only moving this one? It’s the one with pxe traffic on.

File: etc/kayobe/inventory/group_vars/seed/network-interfaces [hunk 2]
```diff
@@ -14,16 +16,16 @@ bond0_net_bond_slaves:
 # Admin network to rest of control plane
 # NOTE: seed node is the default gateway for all other control nodes
 #       via this network, and all other nodes access this via 1GbE
-admin_oc_net_interface: "{{ bond0_net_interface }}.{{ admin_oc_net_vlan }}"
+admin_oc_net_interface: "{{ 1gbe_physical_interface }}.{{ admin_oc_net_vlan }}"
```
Comment (JohnGarbutt) [inline etc/kayobe/inventory/group_vars/seed/network-interfaces]: This provides all compute with the default route, kinda like this on the bond

File: etc/kayobe/kolla.yml [hunk 3]
```diff
@@ -75,7 +75,7 @@ kolla_docker_namespace: rse-cambridge
 #kolla_docker_registry_password:
 
 # Kolla OpenStack release version. This should be a Docker image tag.
-kolla_openstack_release: 7.0.1.9
+kolla_openstack_release: 7.0.1.10
```
Comment (JohnGarbutt) [inline etc/kayobe/kolla.yml:78]: Are you sure about this one, reverts the seed upgrade

**Review by JohnGarbutt — CHANGES_REQUESTED** [review]
I think you are reverting the seed upgrade by accident here.

Personally I would move just the pxe network, will be easier to apply.

File: etc/kayobe/inventory/group_vars/seed/network-interfaces (hunk 2 above)
Comment (paulbrowne) [reply etc/kayobe/inventory/group_vars/seed/network-interfaces]: That's a good point, maybe after we've got these new nodes provisioned we move this back and just keep the provisioning traffic on the 1GbE NIC

File: etc/kayobe/kolla.yml (hunk 3 above)
Comment (paulbrowne) [reply etc/kayobe/kolla.yml:78]: This is fine, I think, as we're incrementing the Kolla release here.

**Review by JohnGarbutt — COMMENTED** [review]
I think this is OK, but just a bit worried about the passwords change, I guess its something new for the seed :/

File: etc/kayobe/kolla.yml (hunk 3 above)
Comment (JohnGarbutt) [reply etc/kayobe/kolla.yml:78]: ah, oops, read that the wrong way around.

**Review by JohnGarbutt — APPROVED** [review]

**Comment by paulbrowne** [comment]
I think we could merge and close this one for now, with an eye to moving the seed to have only the provisioning network on the 1GbE NIC and everything else on the bond in the near future after these new nodes are provisioned.

**Pull request merged**